src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
As far as I remember, spec only specifies the "error" text (in this case "TooManyResults") but "description" is up to the each particular NGSIv2 implementation.

Let's talk about it by skype next daily meeting.